ebtables/ebtables.go

```diff
@@ -51,6 +52,22 @@ func SetArpReply(ipAddress net.IP, macAddress net.HardwareAddr, action string) e
 	return executeShellCommand(command)
 }
 
+// SetEBRule sets an EB rule to route the given IP via the host.
+func SetEBRule(ipAddress string) error {
+	command := fmt.Sprintf(
+		"ebtables -t broute -A BROUTING --ip-dst %s -p IPv4 -j redirect --redirect-target ACCEPT",
+		ipAddress)
+
+	return executeShellCommand(command)
+}
+
+// GetEBRules gets the EB rules to route via host.
+func GetEBRules() (string, error) {
```

Comment:
rename this to GetEbtableRules(tableName, chainName string)

here tableName is broute and chainName is BROUTING